### PR TITLE
Added tap dance to LNG1 key (LNG2)

### PR DESCRIPTION
**Pull request by syntax-magic**
Added tap dance to LNG1 key and turned off RGB Matrix Effects

## Description
- Added tap dance ( LNG2 ) to LNG1 key and ( / ) to  * key  in _NUMPAD layer
- Turned off RGB Matrix Effect by default
- Updated _LEFTFN and _NUMPAD layers
- Updated readme

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/kprepublic/bm40hsrgb/keymaps/dan/config.h [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#pragma once
```
Comment (drashna) [inline keyboards/kprepublic/bm40hsrgb/keymaps/dan/config.h:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment (syntax-magic) [reply keyboards/kprepublic/bm40hsrgb/keymaps/dan/config.h:4]: Hi I have submitted another commit regarding this. There are some shift arounds on keymap but not a major one ( just added tap dance to a numpad layer to access * and / from a single key ). Thanks!

File: keyboards/kprepublic/bm40hsrgb/keymaps/dan/config.h [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// Copyright 2021 Your Name (@syntax-magic)
```
Comment (drashna) [inline keyboards/kprepublic/bm40hsrgb/keymaps/dan/config.h:1]: "Your Name"?  May want to change that. :)

Comment (syntax-magic) [reply keyboards/kprepublic/bm40hsrgb/keymaps/dan/config.h]: Hi drashna, thanks for pointing it out. My name has been added. Cheers.

**Review by drashna — APPROVED** [review]

**Pull request merged**